packages/core/src/order/order-request-sender.ts

```diff
@@ -37,13 +33,15 @@ export default class OrderRequestSender {
             'payments',
             'lineItems.physicalItems.socialMedia',
             'lineItems.physicalItems.options',
+            'lineItems.physicalItems.categories',
             'lineItems.digitalItems.socialMedia',
             'lineItems.digitalItems.options',
+            'lineItems.digitalItems.categories',
         ];
 
         return this._requestSender.get(url, {
             params: {
-                include: joinIncludes([...include, ...((params && params.include) || [])]),
+                include: joinIncludes([...include]),
```

Comment:
```suggestion
                include,
```
It appears that a merging is no longer needed.

Reply:
@bc-peng,
I have done so in the first place, but this function actually joins the array into comma separated string, so it is still needed

Reply:
Oh, I see🤔. The unnecessary merging is this one.

```suggestion
                include: joinIncludes(include),
```

Reply:
Good point, thanks 👍 
Updated